app.js

```diff
@@ -77,33 +78,47 @@ PatCookiesLocation.prototype.render = function() {
   cookieTable.appendChild(trEl);
 };
 
-  // for(var i = 0; i < hours.length; i++) {
-  //   var listItem = document.createElement('li');
-  //   listItem.textContent = hours[i] + ': ' + Math.round(this.hourlySalesArray[i]) + ' cookies';
-  //   var patCookieLocationList = document.getElementById(this.iDPat);
-  //   document.getElementById(this.iDPat).appendChild(listItem);
-  // }
-  // var totalListItem = document.createElement('li');
-  // totalListItem.textContent = 'Total: ' + Math.round(this.totalSales) + ' cookies';
-  // patCookieLocationList.appendChild(totalListItem);
-// };
-
-// All instances and calls of prototypes to calculate hourly customers, hourly sales, and then render on the page
+var wholeTable = function() {
+  makeHeaderRow();
+  for(var i = 0; i < cookieStandsArray.length; i++) {
+    cookieStandsArray[i].render();
+    console.log(cookieStandsArray[i]);
+  }
+};
+// pushed carts into array
 var firstAndPike = new PatCookiesLocation('First and Pike', 23, 65, 6.3, 'pikeID');
-firstAndPike.render();
-
 var seaTac = new PatCookiesLocation('SeaTac', 3, 24, 1.2, 'seaTacID');
-seaTac.render();
-
 var seaCenter = new PatCookiesLocation('Seattle Center', 11, 38, 3.7, 'seaCenterID');
-seaCenter.render();
-
 var capHill = new PatCookiesLocation('Capitol Hill', 20, 38, 2.3, 'capHillID');
-capHill.render();
-
 var alki = new PatCookiesLocation('Alki', 2, 16, 4.6, 'alkiID');
-alki.render();
-
-// Instead of rendering each individually, add each instance to an array (like allStoresArray) and create a function that generates sales numbers by calling the array. This call requires a for loop that calls allStoresArray[i].render(i); Then, call the 'generate sales numbers' function directly below makeHeaderRow at the bottom of the document
-
-// to color alternating lines, add ID to alternating trEL
+//appended to table
+wholeTable();
+//grabbing form
+var cookieForm = document.getElementById('cookie_form');
+//this is our cookie handler
+function handleCookieForm(event) {
+  event.preventDefault();
+  // console.log('I heard a click');
+  var locationName = document.getElementById('location_name');
+  var minCust = document.getElementById('min_cust');
+  var maxCust = document.getElementById('max_cust');
+  var avgCookies = document.getElementById('avg_cookies');
+  var address = document.getElementById('add_address');
+  var phoneNum = document.getElementById('phone_num');
```

Comment:
So let's not forget, there's two ways you can harvest data from your forms. One is to do it the way you have right here. The other way is to access the 'submit' event itself. `event.target.<input field name>.value`. Both are valid but the latter will apply only to the form itself.